sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldNotHaveIdenticalImplementations.cs

```diff
@@ -96,5 +94,33 @@ string Qux2(int val)
         }
 
         string Qux3(int val) => val.ToString(); // Compliant we ignore expression body
+
+        public class Foo
+        {
+            public void Test(string str)
+            {
+                Console.WriteLine(str);
+            }
+        }
+
+        public class Bar
+        {
+            public void Test(string str)
+            {
+                throw new Exception(str);
+            }
+        }
+
+        public static void TestFoo(Foo x)
+        {
+            x.Test("hello");
+            x.Test("world");
+        }
+
+        public static void TestBar(Bar x)
+        {
+            x.Test("hello");
+            x.Test("world");
+        }
```

Comment:
Please, write tests with more than one argument and tests with different number of arguments.